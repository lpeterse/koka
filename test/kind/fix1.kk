type fix<f> {
  con Inn( fix: f<fix<f>> )
}
