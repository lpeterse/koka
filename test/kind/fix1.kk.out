kind/fix1/fix: (V -> V) -> V